src/app/components/VisuallyHiddenText/index.test.tsx

```diff
@@ -16,4 +16,13 @@ describe('VisuallyHiddenText', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('should render off screen text for screen readers as an h1 with lang as en-GB', () => {
```

Comment:
Based on the comments below https://github.com/bbc/simorgh/pull/10762/files#r1159632166 is it worth adding another test for where the VisuallyHiddenText is an anchor tag with a href? Or is that overkill?

Reply:
Think we may not need it but could add it if you like? If removing the snapshots later down the line could be useful. Having the strongly type-checked component this time around might eliminate the need for this particular unit test